codegens/csharp-restsharp/lib/util.js

```diff
@@ -1,19 +1,76 @@
 /**
  * sanitizes input string by handling escape characters eg: converts '''' to '\'\'' and trim input if required
- * 
+ *
  * @param {String} inputString - Input String to sanitize
  * @param {Boolean} [trim] - Indicates whether to trim string or not
  * @returns {String} Sanitized String handling escape characters
  */
 function sanitize (inputString, trim) {
     /* istanbul ignore next */
     if (typeof inputString !== 'string') {
-        throw new Error(`Csharp-Restsharp-Converter:function expects input to be string, found ${typeof inputString}`);
+        return '';
     }
     inputString = inputString.replace(/\\/g, '\\\\').replace(/"/g, '\\"');
     return trim ? inputString.trim() : inputString;
 }
 
+/**
+ * sanitize input options
+ *
+ * @param {Object} options - Options provided by the user
+ * @param {Object} expectedOptions - Expected options built from getOptions
```

Comment:
the term `expected` only makes sense in tests

let's call this `options, defaultOptions`